src/token/erc20/interface.cairo

```diff
@@ -32,9 +32,6 @@ trait IERC20SafeAllowance<TState> {
 
 #[starknet::interface]
 trait IERC20Camel<TState> {
```

Comment:
if we remove these methods, we should rename it to `IERC20CamelOnly` for consistency with other traits

Reply:
We already have `IERC20CamelOnly` just underneath this interface

Reply:
nevermind. reading this again, i don't know what i was thinking